src/genet/schedule_elements.py

```diff
@@ -3697,28 +3698,38 @@ def write_to_json(self, output_dir: str):
             json.dump(self.to_json(), outfile)
         self.write_extras(output_dir)
 
-    def write_to_geojson(self, output_dir: str, epsg: str):
-        """Writes Schedule graph to nodes and edges geojson files.
+    def write_spatial(self, output_dir, epsg: str = None, filetype: Optional[str] = "parquet"):
```

Comment:
```suggestion
    def write_spatial(self, output_dir, epsg: str = None, filetype: str = "parquet"):
```

`Optional` is used when you allow the value to be `None` (i.e., it's equivalent to `str | None`).